uobtheatre/bookings/models.py

```diff
@@ -719,18 +728,29 @@ def seat_price(self) -> int:
             seat_group=self.seat_group
         ).price
 
-    def check_in(self):
+    @property
+    def checked_in(self) -> bool:
+        """Boolean property for if a ticket is checked in.
+
+        Returns:
+            bool: Whether the ticket is checked in.
+        """
+        return self.checked_in_at is not None
```

Comment:
Do we need this? If `checked_in_at` has a timestamp, it will evaluate as truthy, if it doesn't it will evaluate as falsey.

I.e. we can do `if ticket.checked_in_at:` and have the same effect